scripts/sysman/events.yml

```diff
@@ -44,6 +44,8 @@ etors:
       desc: "Event is triggered when accelerator RAS uncorrectable errors cross thresholds (use $sRasSetConfig() to configure - disabled by default)."
     - name: DEVICE_RESET_REQUIRED
       desc: "Event is triggered when the device needs to be reset (use $sDeviceGetState() to determine the reasons for the reset)."
+    - name: SURVIVABILITY_MODE_DETECTED
+      desc: "Event is triggered when graphics driver encounter error condition and Yellow Bang (in Window)."
```

Comment:
After the `desc:` add versioning.

`version: 1.9`

Also, the Spec is platform agnostic. Remove platform-specific language, or rephrase to not exclude any particular platform(s).

i.e. `Event is triggered when the driver encounters an error condition.`